pkg/cli/admin/release/info.go

```diff
@@ -1987,6 +2035,306 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
 	return nil
 }
 
+func (o *InfoOptions) listRpmdb(releaseInfo *ReleaseInfo, dir, format, targetImage string) error {
+	if targetImage == "" {
+		// find image for machine-os component
+		if tag, ok := releaseInfo.ComponentTags["machine-os"]; !ok {
```

Comment:
Is this the image both before and after openshift/enhancements#1637 or when would `rhel-coreos` be used?

Reply:
Yeah, nothing changes in that respect with https://github.com/openshift/enhancements/pull/1637; the node image (`rhel-coreos`/`stream-coreos`) still declares the `machine-os` component and version.

This code works identically before and after https://github.com/openshift/enhancements/pull/1637.